publisher.py

```diff
@@ -0,0 +1,40 @@
+import ftplib
+from ftplib import FTP
+from multiprocessing import Pool
+from io import  BytesIO
+import os
+
+import requests
+
+s = requests.Session()
+
+def retrieve_and_post(match):
+    s = requests.Session()
+    ftp = FTP()
+    ftp.connect(host='127.0.0.1', port=2121)
+    ftp.login(user='ons', passwd='ons')
+    with BytesIO() as sf:
+        s.post('http://0.0.0.0:8080/upload/bres/1/{}'.format(match), data=sf)
+        ftp.retrbinary('RETR %s' %match, sf.write)
+    ftp.close()
+
+
+def main():
```

Comment:
Runs on a background thread on an interval. The interval is currently hardcoded for the spike, but could be set using an env var in production.